src/makeRobotAccountant.js

```diff
@@ -4,9 +4,26 @@
  *
  * @return {function}
  */
-
 function makeRobotAccountant() {
-  // write code here
-}
+  let session = 0;
+
+  const calcSum = function(number) {
```

Comment:
```suggestion
  const calcSum = (number) => {
```
You can make it an arrow function too